From 1e58cda71570582f4283bd37c71e8c846c77c379 Mon Sep 17 00:00:00 2001
From: Florin Chiculita <florinlaurentiu.chiculita@nxp.com>
Date: Tue, 11 Jun 2019 23:53:45 +0300
Subject: [PATCH 221/235] arm64: dts: lx2160a: update interrupt property for
 Aquantia phy

Update Aquantia AQR107 nodes interrupt property.

Signed-off-by: Florin Chiculita <florinlaurentiu.chiculita@nxp.com>
[Xulin: Original patch taken from NXP LSDK-19.06.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts b/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts
index 0f01bfd5e74e..3e46129049cf 100644
--- a/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-lx2160a-rdb.dts
@@ -186,13 +186,13 @@
 	aquantia_phy1: ethernet-phy@4 {
 		/* AQR107 PHY - "compatible" property not strictly needed */
 		compatible = "ethernet-phy-ieee802.3-c45";
-		interrupts = <GIC_SPI 1 IRQ_TYPE_LEVEL_HIGH>;
+		interrupts = <GIC_SPI 2 IRQ_TYPE_LEVEL_HIGH>;
 		reg = <0x4>;
 	};
 	aquantia_phy2: ethernet-phy@5 {
 		/* AQR107 PHY - "compatible" property not strictly needed */
 		compatible = "ethernet-phy-ieee802.3-c45";
-		interrupts = <GIC_SPI 2 IRQ_TYPE_LEVEL_HIGH>;
+		interrupts = <GIC_SPI 3 IRQ_TYPE_LEVEL_HIGH>;
 		reg = <0x5>;
 	};
 };
-- 
2.17.1

